extensions/wikia/FounderEmails/events/FounderEmailsCompleteDigestEvent.class.php

```diff
@@ -27,10 +27,15 @@ public function process ( Array $events ) {
 		$wgTitle = Title::newMainPage();
 		$founderEmailObj = FounderEmails::getInstance();
 		// Get list of founders with digest mode turned on (defined in FounderEmailsEvent)
-		$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		if ( empty( $events ) ) {
+			$cityList = $founderEmailObj->getFoundersWithPreference( 'founderemails-complete-digest' );
+		} else {
+			$cityList = $events;
+		}
 		$wikiService = new WikiService();
 
 		foreach ( $cityList as $cityID ) {
+			Wikia::initAsyncRequest( $cityID );
```

Comment:
Is it necessary to call `Wikia::initAsyncRequest` here? It looks like the email controller [takes care of pulling the correct wikiname form the wiki id](https://github.com/Wikia/app/blob/dev/extensions/wikia/Email/Controller/FounderDigestController.class.php#L25-L26)

The fix is that we weren't passing the wikiId [here](https://github.com/Wikia/app/pull/8027/files#diff-77c4dcf675364821f0c6836036116805R101), right?